pkgs/development/libraries/qt-5/5.4/default.nix

```diff
@@ -152,7 +152,7 @@ let
 
       multimedia = callPackage
         (
-          { qtSubmodule, base, declarative
+          { qtSubmodule, base, declarative, pkgconfig
```

Comment:
Do we gain anything from disabling it by default?  The closure size of qt5.multimedia only increases by 27M to 666M with this change.